add_up.rb

```diff
@@ -0,0 +1,14 @@
+def add_up(i)
```

Comment:
It is better to define a variable here to store the sum of the numbers like -
`sum = 0`